pkg/config/datafileprojectconfig/mappers/integration.go

```diff
@@ -0,0 +1,37 @@
+/****************************************************************************
+ * Copyright 2022 Optimizely, Inc. and contributors                         *
+ *                                                                          *
+ * Licensed under the Apache License, Version 2.0 (the "License");          *
+ * you may not use this file except in compliance with the License.         *
+ * You may obtain a copy of the License at                                  *
+ *                                                                          *
+ *    https://www.apache.org/licenses/LICENSE-2.0                           *
+ *                                                                          *
+ * Unless required by applicable law or agreed to in writing, software      *
+ * distributed under the License is distributed on an "AS IS" BASIS,        *
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. *
+ * See the License for the specific language governing permissions and      *
+ * limitations under the License.                                           *
+ ***************************************************************************/
+
+// Package mappers ...
+package mappers
+
+import (
+	datafileEntities "github.com/optimizely/go-sdk/pkg/config/datafileprojectconfig/entities"
+	"github.com/optimizely/go-sdk/pkg/entities"
+)
+
+// MapIntegrations maps the raw datafile integration entities to SDK Integration entities
+func MapIntegrations(integrations []datafileEntities.Integration) (integrationsList []entities.Integration) {
```

Comment:
This is not a map, simple list either return `map` along with `list` or no need of this method.